src/settings/FileExtensions/ViewFileExtension.js

```diff
@@ -42,6 +49,12 @@ export class ViewFileExtension extends Component {
     onClose: PropTypes.func.isRequired,
   };
 
+  constructor(props) {
+    super(props);
+
+    this.cViewMetaData = props.stripes.connect(ViewMetaData);
```

Comment:
```suggestion
    const { stripes } = this.props;

    this.connectedViewMetaData = stripes.connect(ViewMetaData);
```
Can you rewrite it like this for consistency. Like in Home and DataImportSettings components

Reply:
Refactored